AudioExample/AudioExample.js

```diff
@@ -118,7 +118,7 @@ class AudioExample extends Component {
       }
 
       try {
-        await AudioRecorder.startRecording();
+        await AudioRecorder.resumeRecording();
```

Comment:
I don't see a definition for `AudioRecorder.resumeRecording()` on `index.js`. This seems to be braking the master branch.